src/services/textChanges.ts

```diff
@@ -64,6 +64,13 @@ namespace ts.textChanges {
         return skipTrivia(text, start, /*stopAfterLineBreak*/ false, /*stopAtComments*/ true);
     }
 
+    function rangeOfTypeParameters(text: string, typeParameters: NodeArray<TypeParameterDeclaration>): TextRange {
+        // Include the `<>`
+        const pos = typeParameters.pos - "<".length;

```

Comment:
when is `"<".length !== 1` ?

Reply:
<img width="601" alt="Screenshot 2020-09-08 at 21 47 05" src="https://user-images.githubusercontent.com/509265/92515797-dbde3380-f21c-11ea-9a09-f052c810b21f.png">
Seems this question related to this code too :).